pkg/rtc/mediatrackreceiver.go

```diff
@@ -117,7 +118,9 @@ func NewMediaTrackReceiver(params MediaTrackReceiverParams, ti *livekit.TrackInf
 		params: params,
 		state:  mediaTrackReceiverStateOpen,
 	}
-	t.trackInfo.Store(proto.Clone(ti).(*livekit.TrackInfo))
+	tic := proto.Clone(ti).(*livekit.TrackInfo)
+	tic.PublishedAt = time.Now().UnixMilli()
+	t.trackInfo.Store(tic)
```

Comment:
We already have a `TimedVersion` for the track. Can that be used?
That one does not change for migrated track, but does change for a re-published track (which retains the `trackID`).
This one changes for both (migrated and re-published). Is that the intent?

Reply:
nope, not the intent to change on migration. will use the timestamp from version.